lib/tests/fixtures/ruby/uses_exit_code.rb

```diff
@@ -0,0 +1,7 @@
+Facter.add(:foo) do
+  setcode do
+    cmd = Gem.win_platform? ? 'cmd /k "exit 99"' : 'sh -c "exit 99"'
```

Comment:
This seemed like the most expedient way to find out if we were windows-y, but if there's another quick way I'm happy to tweak this line. the 'Gem' module exists in all rubies 1.9+